src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java

```diff
@@ -0,0 +1,114 @@
+package org.folio.rest.impl;
+
+import static org.folio.oaipmh.Constants.OKAPI_TENANT;
+import static org.folio.oaipmh.Constants.REPOSITORY_FETCHING_CLEAN_ERRORS_INTERVAL;
+import static org.folio.rest.jaxrs.resource.OaiPmhCleanUpInstances.PostOaiPmhCleanUpInstancesResponse.respond500WithTextPlain;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.oaipmh.service.ErrorsService;
+import org.folio.oaipmh.service.InstancesService;
+import org.folio.rest.jaxrs.resource.OaiPmhCleanUpErrorLogs;
+import org.folio.rest.jaxrs.resource.OaiPmhCleanUpInstances;
+import org.folio.s3.client.FolioS3Client;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.core.Response;
+import java.io.File;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class CleanUpErrorLogs implements OaiPmhCleanUpErrorLogs {
+
+  private final Logger logger = LogManager.getLogger(this.getClass());
+
+  @Autowired
+  private FolioS3Client folioS3Client;
+
+  @Autowired
+  private InstancesService instancesService;
+
+  @Autowired
+  private ErrorsService errorsService;
+
+  public CleanUpErrorLogs() {
+    SpringContextUtil.autowireDependencies(this, Vertx.currentContext());
+  }
+
+  @Override
+  public void postOaiPmhCleanUpErrorLogs(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    logger.debug("Running cleaning up error logs");
+    final long[] cleanInterval = {30};
+
+    ModTenantAPI api = new ModTenantAPI();
+    List<String> configsSet = Arrays.asList("technical");
+
+    api.loadConfigurationData(okapiHeaders, configsSet).onComplete(asyncResult -> {
+      if (asyncResult.succeeded()) {
+        try {
+          cleanInterval[0] = Long.parseLong(System.getProperty(REPOSITORY_FETCHING_CLEAN_ERRORS_INTERVAL));
+        } catch (Exception ex) {
+          logger.error("cannot retrieve config from system properties");
+        }
+      } else {
+        logger.error("loading configuration data failed");
+      }
+    });
+
+    OffsetDateTime offsetDateTime = ZonedDateTime
+      .ofInstant(Instant.now(), ZoneId.systemDefault())
```

Comment:
Wouldn't be better to use UTC instead of system default here?

Reply:
fixed